plugins/inputs/t128_lte/t128_lte.go

```diff
@@ -208,28 +212,21 @@ func (plugin *T128Lte) loadNodeName() (string, error) {
 	return localData.Init.Id, nil
 }
 
-func (plugin *T128Lte) getLteInterfaces() <-chan lteDeviceInterface {
-	lteInterfaces := make(chan lteDeviceInterface)
-	content, err := readT128Config(plugin.T128ConfigFile)
+func (plugin *T128Lte) getLteInterfaces() []lteDeviceInterface {
+	content, err := readT128Config(plugin.t128ConfigFile)
 	if err != nil {
-		return lteInterfaces
+		return []lteDeviceInterface{}
 	}
-	node, err := matchNode(plugin.NodeName, content)
+	node, err := matchNode(plugin.nodeName, content)
 	if err != nil {
 		fmt.Errorf("Unable to find node name %s : %w", node, err)
-		return lteInterfaces
+		return []lteDeviceInterface{}
 	}
 	deviceInterfaces, err := plugin.getDeviceInterface(node, content)
```

Comment:
We already parsed the config for `matchNode`, you should pass the parsed struct here. Don't waste time by parsing the content again...